gtm_template.tpl

```diff
@@ -1777,13 +1779,15 @@ const acceptedStorageNames = [];
 for (let key in storagePool) {
   const storage = storagePool[key];
   
-  let defaultStorageConsent = -1 !== consentCookie.indexOf(storage.name) || storage.enabled_by_default ? 'granted' : 'denied';
+    let defaultStorageConsent = ((null === consentCookie && storage.enabled_by_default) || (null !== consentCookie && -1 !== consentCookie.indexOf(storage.name))) ? 'granted' : 'denied';
```

Comment:
Alignment's a little off

Reply:
Fixed in https://github.com/68publishers/cookie-consent/pull/55